pipelines/A14/queries.sql

```diff
@@ -56,34 +56,35 @@ SELECT T.total_full_time_employment_of_tourism
 FROM tourism_economy_by_canton as T
 JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid
 WHERE S.country=TRUE
-    AND S.name_de = "Schweiz"
+    AND S.name_de = 'Schweiz'
     AND T.year = 2019;
 
 -- What percentage of full time employment was allocated in canton Zurich between 2017 and 2019?
 SELECT T.year, T.percent_share_full_time_employment_of_tourism as percent_share_fte_tourism_canton_zurich
 FROM tourism_economy_by_canton as T
 JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid
 WHERE S.canton = TRUE
-    AND S.name_de = "Kanton Zürich"
+    AND S.name_de = 'Kanton Zürich'
```

Comment:
```suggestion
    AND S.name LIKE '%Zürich%'
```
Maybe we should always match on `S.name` and not `S.name_de`, especially for english-language datasets. Also using `LIKE` match for consistency